os/dbuild.sh

```diff
@@ -84,6 +84,8 @@ function FIND_BINFILE()
 			BINFILE="${BINDIR}/tinyara_master_strip"
 		elif [[ "${CONFIG_ARCH_BOARD}" == "rtl8721csm" ]]; then
 			BINFILE="${BINDIR}/${KERNEL_BIN_NAME}"
+		elif [[ "${CONFIG_ARCH_BOARD}" == "rtl8720e" ]]; then
+			BINFILE="${BINDIR}/${KERNEL_BIN_NAME}"
```

Comment:
BINFILE name is same, you can put in || condition with rtl8721csm

Reply:
We are modifying this function. Let's merge it as it is.